storage/postgres.go

```diff
@@ -1,7 +1,45 @@
-// Package storage provides implementation for persisting user information.
-// It includes the following methods:
-// - create new user
-// - get information about exists user
-// - update exists user
-// - delete unnecessary user
+// Package storage has to provide an interface to create a new postgres client
+// with ability to reconnect after connection failure.
 package storage
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	// _ mean that we can use all function from this package
+	_ "github.com/lib/pq"
+)
+
+// Config of Postgres DB
+type PgClient struct {
```

Comment:
WDYT about renaming PgClient to DBConfig or Config

Reply:
Agree, that's better name. Changed it.